externs/browser/w3c_event.js

```diff
@@ -103,20 +103,19 @@ function Event(type, opt_eventInitDict) {}
  * @type {number}
  * @see http://www.w3.org/TR/DOM-Level-2-Events/ecma-script-binding.html
  */
-Event.AT_TARGET;
+Event.CAPTURING_PHASE = 1;
```

Comment:
The compiler only records type information from externs files.
Assigning a value here is misleading, because the compiler will totally ignore it.